mne/viz/topomap.py

```diff
@@ -965,7 +965,7 @@ def _plot_ica_topomap(ica, idx=0, ch_type=None, res=64, layout=None,
                       vmin=None, vmax=None, cmap='RdBu_r', colorbar=False,
                       title=None, show=True, outlines='head', contours=6,
                       image_interp='bilinear', head_pos=None, axes=None,
-                      sensors=True, allow_ref_meg=False):
+                      sensors=True, allow_ref_meg=False, extrapolate='box'):
```

Comment:
is extrapolate='box' the default so this does not change any current behavior?

Reply:
yes: https://github.com/mne-tools/mne-python/blob/master/mne/viz/topomap.py#L628

Reply:
yes, 'box' is the old behavior